runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -38,8 +38,13 @@
      * The Transaction stream is used to log/write successful transactions from different clients.
      * Transaction data and meta data can be obtained by reading this stream.
      */
+    @Deprecated
     public static final UUID TRANSACTION_STREAM_ID = CorfuRuntime.getStreamID("Transaction_Stream");
 
+    // We are temporarily naming this stream with the same name as TRANSACTION_STREAM_ID to avoid breaking LR code
+    // while transition to UFO is completed (once migration is complete to UFO this stream can be renamed)
+    public static final UUID LOG_REPLICATOR_STREAM_ID = CorfuRuntime.getStreamID("Transaction_Stream");
```

Comment:
I think the probability that one day someone creates a transaction_stream table is pretty high, let's add some prefix, for instance: `org.corfudb.logreplication.transactionstream`

Reply:
I get your suggestion and we can definitely do it on a second round, I've added it to the TODO, the reason why I don't want to do it now is because this most likely is used by DAO and if we change it here, any code still not migrated to UFO will break.